### PR TITLE
Add WhiteOctoberPagerfantaBundle

### PR DESCRIPTION
**Pull request by mikadamczyk**
Add WhiteOctoberPagerfantaBundle to implement pagination feature in new Admin UI

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
         "willdurand/js-translation-bundle": "^2.6.4",
         "oneup/flysystem-bundle": "^2.0",
         "friendsofsymfony/jsrouting-bundle": "^1.4",
-        "roave/security-advisories": "dev-master"
+        "roave/security-advisories": "dev-master",
+        "white-october/pagerfanta-bundle": "^1.1"
```
Comment (andrerom) [inline composer.json]: could you add it before instead, simplifies rebases :)

File: app/AppKernel.php [hunk 2]
```diff
@@ -37,10 +37,10 @@ public function registerBundles()
             new EzSystems\PlatformInstallerBundle\EzSystemsPlatformInstallerBundle(),
             new EzSystems\RepositoryFormsBundle\EzSystemsRepositoryFormsBundle(),
             new EzSystems\EzPlatformSolrSearchEngineBundle\EzSystemsEzPlatformSolrSearchEngineBundle(),
-            // new EzSystems\EzPlatformDesignEngineBundle\EzPlatformDesignEngineBundle(),
             new EzSystems\EzPlatformAdminUiBundle\EzPlatformAdminUiBundle(),
             new EzSystems\EzPlatformAdminUiModulesBundle\EzPlatformAdminUiModulesBundle(),
             new EzSystems\EzPlatformAdminUiAssetsBundle\EzPlatformAdminUiAssetsBundle(),
+            new WhiteOctober\PagerfantaBundle\WhiteOctoberPagerfantaBundle(),
```
Comment (adamwojs) [inline app/AppKernel.php]: IMHO this should be placed under "Dependencies" section

File: app/AppKernel.php [hunk 3]
```diff
@@ -37,10 +37,10 @@ public function registerBundles()
             new EzSystems\PlatformInstallerBundle\EzSystemsPlatformInstallerBundle(),
             new EzSystems\RepositoryFormsBundle\EzSystemsRepositoryFormsBundle(),
             new EzSystems\EzPlatformSolrSearchEngineBundle\EzSystemsEzPlatformSolrSearchEngineBundle(),
-            // new EzSystems\EzPlatformDesignEngineBundle\EzPlatformDesignEngineBundle(),
```
Comment (andrerom) [inline app/AppKernel.php]: maybe skip removing this, we will, just like this one, probably re add it anyway, so good to keep track of what was disabled / removed on the initial admin ui import ;)

File: composer.json (hunk 1 above)
Comment (mikadamczyk) [reply composer.json]: ok :)

**Comment by mikadamczyk** [comment]
Fixed :)

**Pull request merged**